gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -145,6 +145,14 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
+    parser.add_argument(
+        '--schema_version',
+        type=int, default=0,
```

Comment:
nit: how about making default 1 and the new one 2?

Even better would be to use a proper enum for this, rather than just digits - like for example 'Schema.V1' and 'Schema.V2' or something of that sort.